api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -2050,18 +2050,8 @@ private void enableEpilepsyProgram() {
                 )),
                 "coreapps", "dashboardwidgets/dashboardWidget"));
 
-        apps.add(addToEpilepsyDashboardSecondColumn(app(Apps.EPILEPSY_SEIZURES,
-                "pih.app.epilepsy.seizureGraph",
-                "icon-bar-chart",
-                null,
-                null,
-                objectNode(
-                        "widget", "obsgraph",
-                        "label", "pih.app.epilepsy.seizureGraph",
-                        "icon", "icon-bar-chart",
-                        "conceptId", MirebalaisConstants.EPI_SEIZURES,
-                        "maxResults", "30"  // MEX-127
```

Comment:
MEX-127 is actually unrelated; not sure why it's referenced here. It has to do with Obs Across Encounters.